package.json

```diff
@@ -59,7 +59,7 @@
     "systemjs-builder": "^0.15.13",
     "through2": "^2.0.1",
     "tslint": "3.7.0-dev.5",
-    "typescript": "1.9.0-dev.20160505",
+    "typescript": "^2.0.9",
```

Comment:
do you think we should keep this version fixed? i remember last time there was an issue when we upgraded a typescript version. not sure what it was? cc: @youdz, @jeeyun